### PR TITLE
Fix the password urls and forms

### PR DESCRIPTION
**Pull request by PierrePaul**
Overwrites the url name to make sure the proper form is loaded and the proper url is used.

https://trello.com/c/pI9pKUN8/304-make-sure-that-the-forms-inherit-properly

### CODE REVIEW

File: profiles/urls.py [hunk 1]
```diff
@@ -88,6 +88,7 @@
 )
 # Django.contrib.auth urls have underscore in them, let's change that for dashes
 urlpatterns += [
+    path("", include("django.contrib.auth.urls")),
```
Comment (pcraig3) [inline profiles/urls.py:91]: Doesn't going first mean that it has precedence?

Comment (PierrePaul) [reply profiles/urls.py:91]: It does, but then we overwrite the route named 'password_reset' and make it point to our form instead.

**Comment by pcraig3** [comment]
The "new password" form having the colons. We want to get rid of the 'autocomplete' attributes that are in there.

image

**Comment by PierrePaul** [comment]
> The "new password" form having the colons. We want to get rid of the 'autocomplete' attributes that are in there, and inheriting from the base form removes that for us.

Since are overwriting all endpoints of django.contrib.auth.urls, I removed the include. I added a new Form that inherits HealthcareBaseForm to remove the `:` in the Confirm Reset Password view.

**Pull request merged**